v-next/hardhat-build-system/src/internal/solidity/compiler/downloader.ts

```diff
@@ -293,12 +302,12 @@ export class CompilerDownloader implements ICompilerDownloader {
 
   async #shouldDownloadCompilerList(): Promise<boolean> {
     const listPath = this.#getCompilerListPath();
-    if (!(await fsExtra.pathExists(listPath))) {
+    if (!(await exists(listPath))) {
       return true;
     }
 
-    const stats = await fsExtra.stat(listPath);
-    const age = new Date().valueOf() - stats.ctimeMs;
+    const ctime = await getChangeTime(listPath);
```

Comment:
@kanej @schaable spend a few more seconds here just to double check. This change is a bit more "complex"